setup.cfg

```diff
@@ -1,6 +1,6 @@
 [tool:pytest]
 DJANGO_SETTINGS_MODULE = lms.envs.test
-addopts = --nomigrations --reuse-db --durations=20
+addopts = --nomigrations --reuse-db --durations=20 -Wa --json-report
```

Comment:
We're already adding `-Wd` for paver pytest runs to get deprecation warnings, so we may want to skip `-Wa` here for now.  For example: https://github.com/edx/edx-platform/blob/master/pavelib/utils/test/suites/pytest_suite.py#L161 .   Right now I suspect the number of warnings is a bit overwhelming, but we might want to enable this here after we've cleaned up more of them.